<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QDialog" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>510</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Noto Sans Math</family>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    background-color: #3E424E;
}

QLineEdit#lineEdit_2 {
    background-color: #A2A289;
    color: #222;
    font-size: 11pt;
    font-weight: bold;
    font-family: &quot;Noto Sans Math Bold&quot;, monospace;
    padding: 4px;
    border: none;
	border-radius: 8px;
}


QLineEdit#lineEdit {
    background-color: #A2A289;
    color: #222;
    font-size: 18pt;
    font-weight: bold;
    font-family: &quot;Noto Sans Math Bold&quot;, monospace;
    padding: 4px;
    border: none;
	border-radius: 8px;
}

QTextBrowser {
    background-color: #A2A289;
    color: #222;
    font-size: 8pt;
    font-weight: bold;
    font-family: &quot;Noto Sans Math Bold&quot;, monospace;
    padding: 4px;
    border: none;
	border-radius: 8px;
}


/* 모든 QPushButton 공통 스타일 */
QPushButton {
    border: 2px solid #5a5a5a;
    border-radius: 8px;
    padding: 4px;
}

/* 숫자 버튼 */
QPushButton#pushButton_0, QPushButton#pushButton_1, QPushButton#pushButton_2,
QPushButton#pushButton_3, QPushButton#pushButton_4, QPushButton#pushButton_5,
QPushButton#pushButton_6, QPushButton#pushButton_7, QPushButton#pushButton_8,
QPushButton#pushButton_9 {
    background-color: #A0A0A0;
    color: black;
    font-size: 20pt;
    font-weight: bold;
    border-radius: 8px;
    border: 2px solid #5a5a5a;
    padding: 4px;
}

QPushButton#pushButton_0:hover, QPushButton#pushButton_1:hover,
QPushButton#pushButton_2:hover, QPushButton#pushButton_3:hover,
QPushButton#pushButton_4:hover, QPushButton#pushButton_5:hover,
QPushButton#pushButton_6:hover, QPushButton#pushButton_7:hover,
QPushButton#pushButton_8:hover, QPushButton#pushButton_9:hover {
    background-color: #B0B0B0;
}

QPushButton#pushButton_0:pressed, QPushButton#pushButton_1:pressed,
QPushButton#pushButton_2:pressed, QPushButton#pushButton_3:pressed,
QPushButton#pushButton_4:pressed, QPushButton#pushButton_5:pressed,
QPushButton#pushButton_6:pressed, QPushButton#pushButton_7:pressed,
QPushButton#pushButton_8:pressed, QPushButton#pushButton_9:pressed {
    background-color: #888888;
    padding-top: 6px;
    padding-left: 4px;
}


/* 특수 기능 버튼 */
QPushButton#pushButton_point, QPushButton#pushButton_sign,
QPushButton#pushButton_lparen, QPushButton#pushButton_rparen {
    background-color: #7D7E84;
    color: white;
    font-size: 20pt;
    font-weight: bold;
    border-radius: 8px;
    border: 2px solid #5a5a5a;
    padding: 4px;
}

QPushButton#pushButton_point:hover, QPushButton#pushButton_sign:hover,
QPushButton#pushButton_lparen:hover, QPushButton#pushButton_rparen:hover {
    background-color: #8E8F95;
}

QPushButton#pushButton_point:pressed, QPushButton#pushButton_sign:pressed,
QPushButton#pushButton_lparen:pressed, QPushButton#pushButton_rparen:pressed {
    background-color: #5C5D61;
    padding-top: 6px;
    padding-left: 4px;
}


/* 연산자 버튼 */
QPushButton#pushButton_add, QPushButton#pushButton_sub,
QPushButton#pushButton_mul, QPushButton#pushButton_div, QPushButton#pushButton_equal {
    background-color: #7D7E84;
    color: white;
    font-size: 20pt;
    font-weight: bold;
    border-radius: 8px;
    border: 2px solid #5a5a5a;
    padding: 4px;
}

QPushButton#pushButton_add:hover, QPushButton#pushButton_sub:hover,
QPushButton#pushButton_mul:hover, QPushButton#pushButton_div:hover,
QPushButton#pushButton_equal:hover {
    background-color: #8E8F95;
}

QPushButton#pushButton_add:pressed, QPushButton#pushButton_sub:pressed,
QPushButton#pushButton_mul:pressed, QPushButton#pushButton_div:pressed,
QPushButton#pushButton_equal:pressed {
    background-color: #5C5D61;
    padding-top: 6px;
    padding-left: 4px;
}


/* AC, C 버튼 – 기존 색 유지 + 입체감 적용 */
QPushButton#pushButton_AC, QPushButton#pushButton_C {
    background-color: #BA555F;
    color: white;
    font-weight: bold;
    font-size: 20pt;
    border-radius: 8px;
    border: 2px solid #881818;
    padding: 4px;
}

QPushButton#pushButton_AC:hover, QPushButton#pushButton_C:hover {
    background-color: #C76670;
}

QPushButton#pushButton_AC:pressed, QPushButton#pushButton_C:pressed {
    background-color: #98313C;
    padding-top: 6px;
    padding-left: 4px;
}
</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>311</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans Math Bold</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Sans</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>220</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Sans</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>220</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Sans</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Sans</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>280</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Sans</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>280</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Sans</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Sans</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>340</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Sans</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>340</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Sans</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_0">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>400</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Sans</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_lparen">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>31</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Sans</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>( </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_div">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>160</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Sans</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_mul">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>220</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Sans</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>×</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_sub">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>280</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Sans</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_add">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>340</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Sans</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_point">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>400</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Sans</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_equal">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>400</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Sans</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_AC">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Sans</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>AC</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_C">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Sans</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_sign">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Sans</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>±</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_rparen">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>31</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Sans</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>)</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans Math Bold</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
